VERSION

```diff
@@ -1 +1 @@
-1.2.0
+1.2.1
```

Comment:
There is no need to increase the version number, please revert this change.

Reply:
Done, reverted it.